Marlin/pins_MINITRONICS.h

```diff
@@ -123,3 +130,22 @@
 
   #define SD_DETECT_PIN    -1 // Minitronics doesn't use this
 #endif
+
+
+// Rev B additions
+
+//
+//  M3, M4 & M5 support
+//
+#if ENABLED(SPINDLE_ENABLE)  // assumes we're only doing CNC work (no 3D printing)
+  #undef  HEATER_BED_PIN
+  #undef  TEMP_BED_PIN           // need to free up some pins but also need to
+  #undef  TEMP_0_PIN             // re-assign them (to unused pins) because Marlin
+  #undef  TEMP_1_PIN             // requires the presence of certain pins or else it
+  #define HEATER_BED_PIN      4  // won't compile
+  #define TEMP_BED_PIN       50
+  #define TEMP_0_PIN         51
+  #define SPINDLE_ENABLE_PIN 52 // using A6 because it already has a pull up on it
+  #define SPINDLE_SPEED_PIN   3 // WARNING - LED & resistor pull up to +12/+24V stepper voltage
+  #define SPINDLE_DIR_PIN    53
+#endif
```

Comment:
Here, the dummy assignments are kind of a hack, though I get the reasoning. It's a lot more work to make Marlin support `#define EXTRUDERS 0`. But maybe that would be good to support in the long run.